### ISSUE TITLE
Tagging menu is showing the Karel/Candli map in the menu of tags

### ISSUE THREAD

**Issue by OECDPILA** [pila-project/pila-platform#121]
(Medium Priority)

For some reason the candli/karel map is showing up in the menu, even though I was meaning to tag this map with the skills (e.g. repetition and conditionals).

![image](https://github.com/pila-project/pila-platform/assets/145113767/262d46af-b168-4a6b-8097-e6674a57b821)

**Comment by OECDPILA** [comment]
@jasonford @mthomasmcclain - this was the UUID for the mixed Candli and Karel sequence. I removed tagging to specific competencies to fix this for now (seeing as it is a hacked sequence).

**Comment by jasonford** [comment]
@OECDPILA I'll leave this open since I do want to investigate a little bit. Thanks for the additional info!

**Comment by jasonford** [comment]
@OECDPILA I got to the bottom of it!

I accidentally pasted in the wrong uuid when I shared it with you in https://github.com/pila-project/pila-platform/issues/108. Here's the corrected Candli + Karel uuid: 87d1fe60-0998-11ef-9899-83d810e01611

The default behavior of the tag site is to make a tag with the string entered if it is not a UUID. We probably want to fix that/warn if there is a uuid in the string....
